docs/support.md

```diff
@@ -0,0 +1,40 @@
+![MustiXscale logo](img/logos/multixscale_logo.png){width=60%}
+
+Thanks to the MultiXscale project we are able to provide support to the users of EESSI. 
+The support portal is supported by GitLab.
+
+## How to report a problem or ask a question
+
+We recommend you to get a GitLab account for all your support needs.  
+
+If you have a GitLab account you can submit your problems or questions on 
+the following GitLab repo, https://gitlab.com/eessi/support. 
+Please use one of the provided templates (report a problem, software request, question) when submitting an issue.
+
+You can also submit your issues or send in questions via our e-mail address `support (@) eessi.io`. 
+When you send us an email, please provide us with as much information as possible on the issue that you are having. 
+You can find lists on the information that we would like to receive in the ReadMe of the GitLab repository.
+
+## What do we support
+
+### Getting access the EESSI software stack
+If you have trouble connecting to the software stack. 
+Such as trouble related the [cernVM-FS filesystem layer](filesystem_layer.md) or the [compat layers](compatibility_layer.md). 
+Please contact us. 
+
+### software requests
+We are open to software requests for applications that aren't available. You can try it your self by looking through the [documentation on adding software](software_layer/adding_software.md). 
+Or you can just send in a request. 
+Please try to provide as much information on the software as possible.
+
+### Test-suite
+If you are using the test-suite and run into any trouble using it you can also sent them in via our GitLab support portal or email.
+
+### Using the software
+Please report any problems that you encounter when using software in the EESSI stack that are specifically related to the installation within EESSI. 
```

Comment:
This probably needs some more level of detail, like example of what we would support, and something we would not.